gateway-js/src/__tests__/integration/networkRequests.test.ts

```diff
@@ -605,11 +570,15 @@ describe('Downstream service health checks', () => {
       expect(getRootQueryFields(gateway.schema)).toContain('topReviews');
       expect(getRootQueryFields(gateway.schema)).not.toContain('review');
 
-      await schemaChangeBlocker;
+      await errorLoggedPromise;
+      expect(logger.error).toHaveBeenCalledWith(
+        `UplinkFetcher failed to update supergraph with the following error: The gateway subgraphs health check failed. Updating to the provided \`supergraphSdl\` will likely result in future request failures to subgraphs. The following error occurred during the health check:\n[accounts]: 500: Internal Server Error`,
+      );
 
       // At this point, the mock update should have been called but the schema
       // should still be the original.
-      expect(mockUpdateSchema).toHaveBeenCalledTimes(2);
+      expect(updateSpy).toHaveBeenCalledTimes(1);
```

Comment:
double check but I think this is a correct change

Reply:
Yep no reason to notify of an update when update fails, I'd consider this a bug fix